Documentation - Warning missing symbol

/github/workspace/Ribosoft/Models/JobsViewModels/JobIndexViewModel.cs:38: warning: documented symbol 'FileUpload Ribosoft.Models.JobsViewModels.JobIndexViewModel::FormFile' was not declared or defined.

--- a/Doxyfile
+++ b/Doxyfile
@@ -38,7 +38,7 @@ PROJECT_NAME           = "Ribosoft"
 # could be handy for archiving the generated documentation or if some version
 # control system is used.
 
-PROJECT_NUMBER         = "v2.1.0"
+PROJECT_NUMBER         = "v2.1.1"
 
 # Using the PROJECT_BRIEF tag one can provide an optional one line description
 # for a project that appears at the top of each page and should give viewer a

--- a/Ribosoft/Models/JobsViewModels/JobIndexViewModel.cs
+++ b/Ribosoft/Models/JobsViewModels/JobIndexViewModel.cs
@@ -35,7 +35,7 @@ namespace Ribosoft.Models.JobsViewModels
          */
         public List<string> SuccessMessages { get; set; }
 
-        /*! \property FileUpload.FormFile
+        /*! \property UploadFile
          * \brief Holder for uploaded jobs file
          */
         [BindProperty]

--- a/Ribosoft/Views/Shared/_Layout.cshtml
+++ b/Ribosoft/Views/Shared/_Layout.cshtml
@@ -56,7 +56,7 @@
     </div>
 
     <footer class="container">
-        <p class="text-muted">&copy; 2021 - Ribosoft [2.1.0]</p>
+        <p class="text-muted">&copy; 2021 - Ribosoft [2.1.1]</p>
     </footer>
 
     <script src="~/dist/vendor.js" asp-append-version="true"></script>

